### PR TITLE
FRW-8766 Order matrix performance improvement integration.

### PR DESCRIPTION
**Pull request by kraal-spryker**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/FRW-8766

###### Optional

- Core PR: URL_HERE

###### Change log

{Relevant changes for project level go here.}

### CODE REVIEW

File: src/Orm/Zed/KernelApp/Persistence/SpyAppConfigQuery.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+/**
+ * Skeleton subclass for performing query and update operations on the 'spy_app_config' table.
+ *
+ *
+ *
+ * You should add additional methods to this class to meet the
+ * application requirements. This class will only be generated as
+ * long as it does not already exist in the output directory.
+ */
+class SpyAppConfigQuery extends BaseSpyAppConfigQuery
```
Comment (annakotiuk) [inline src/Orm/Zed/KernelApp/Persistence/SpyAppConfigQuery.php:21]: Do you need this?

File: src/Orm/Zed/KernelApp/Persistence/SpyAppConfig.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+/**
+ * Skeleton subclass for representing a row from the 'spy_app_config' table.
+ *
+ *
+ *
+ * You should add additional methods to this class to meet the
+ * application requirements. This class will only be generated as
+ * long as it does not already exist in the output directory.
+ */
+class SpyAppConfig extends BaseSpyAppConfig
```
[the remark this answers is not in the document]
Comment (kraal-spryker) [reply src/Orm/Zed/KernelApp/Persistence/SpyAppConfig.php:21]: it comes with spryker-feature/spryker-core update.

File: src/Orm/Zed/KernelApp/Persistence/SpyAppConfigQuery.php (hunk 1 above)
Comment (kraal-spryker) [reply src/Orm/Zed/KernelApp/Persistence/SpyAppConfigQuery.php:21]: it comes with spryker-feature/spryker-core update.

Comment (kraal-spryker) [reply src/Orm/Zed/KernelApp/Persistence/SpyAppConfigQuery.php:21]: b2c have it, marketplace does not have

Comment (kraal-spryker) [reply src/Orm/Zed/KernelApp/Persistence/SpyAppConfigQuery.php:21]: maybe this feature is not a part of marketplace

**Review by annakotiuk — APPROVED** [review]

**Review by asmarovydlo — APPROVED** [review]

**Pull request merged**